eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -120,6 +119,7 @@ import Lens.Micro ((%~), (&), (+~), (.~), (<>~), (^.))
 data ConwayEpochEvent era
   = PoolReapEvent (Event (EraRule "POOLREAP" era))
   | SnapEvent (Event (EraRule "SNAP" era))
+  | EpochBoundaryRatifyState (RatifyState era)
```

Comment:
Could you please list this addition in the changelog here: https://github.com/input-output-hk/cardano-ledger/blob/6d30ca25215a6dfaccbea95a2e1f89046ed455d9/eras/conway/impl/CHANGELOG.md?plain=1#L4